signbank/static/js/gloss_edit.js

```diff
@@ -255,11 +255,16 @@ function configure_edit() {
 
      $('.edit_list').on('click', function() {
          var choices = choice_lists[$(this).attr('id')];
+         var current_list = $(this)[0].id;
+         if (current_list == 'signer') {
+             choices[''] = 'None';
```

Comment:
@Dimithri up to you, but I've intentionally used a more placeholder value here, just in case NZSL end up using 'None' as a value. 


```suggestion
             choices[''] = '--None--';
```

You should probably see if you can add this as the first item of the object as well. I'm not sure how to do that, but it's a good convention to stick to to show the 'default' or placeholder value first.

Reply:
Actually one easy way to do this is with `$.extend` (or destructuring, but we can't really write ES6 here and assume it'll work). Try `choices = $.extend({'': '--None--'}, choices)` and see if it comes out in the right order.